frb/frb.py

```diff
@@ -185,6 +189,39 @@ def set_ee(self, a, b, theta, cl, stat=True):
         #
         return
 
+    @property
+    def sig_a(self):
+        """
+        Combined semi-major axis error
+
+        Returns:
+            float:
+
+        """
+
+        if len(self.eellipse) == 0:
+            return None
+        siga = self.eellipse['a']  # arcsec
+        if 'a_sys' in self.eellipse.keys():
+            siga = np.sqrt(self.eellipse['a_sys']**2 + siga**2)
+        return siga
+
+    @property
+    def sig_b(self):
+        """
+        Combined semi-minor axis error
+
+        Returns:
+            float:
+
+        """
+        if len(self.eellipse) == 0:
+            return None
+        sigb = self.eellipse['b']  # arcsec
+        if 'b_sys' in self.eellipse.keys():
+            sigb = np.sqrt(self.eellipse['b_sys']**2 + sigb**2)
```

Comment:
Similar to the above. The resultant ellipse should be a convolution of the two Gaussians. On a related note, there should be a method that outputs the PA of the resultant convolution.

Reply:
same